Document aws/v0.11.0 (#1368)
grafana/k6-docs: src/data/markdown/docs/20 jslib/01 jslib/01 aws/SQSClient/00 sendMessage.md

| :------------ | :---------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `queueUrl`    | string            | The URL of the Amazon SQS queue to which a message is sent. Queue URLs and names are case-sensitive.                                                                                                                  |
| `messageBody` | string            | The message to send. The minimum size is one character. The maximum size is 256 KB.                                                                                                                                   |
| `options`     | [SendMessageOptions](#sendmessageoptions) (optional) | Options for the request. |

#### SendMessageOptions

| Name                      | Type   | Description                                                                                     |
| :------------------------ | :----- | :---------------------------------------------------------------------------------------------- |

> | `messageDeduplicationId`  | string (optional) | his parameter applies only to FIFO (first-in-first-out) queues. The token used for deduplication of sent messages. If a message with a particular MessageDeduplicationId is sent successfully, any messages sent with the same MessageDeduplicationId are accepted successfully but aren't delivered during the 5-minute deduplication interval. |

Reviewer: ```suggestion
| `messageDeduplicationId`  | string (optional) | The token used for deduplication of sent messages. This parameter applies only to FIFO (first-in-first-out) queues. If a message with a particular MessageDeduplicationId is sent successfully, any messages with the same MessageDeduplicationId are accepted but not delivered during the 5-minute deduplication interval. |
```

> | `messageGroupId`          | string (optional) | The tag that specifies that a message belongs to a specific message group. Messages that belong to the same message group are processed in a FIFO manner (however, messages in different message groups might be processed out of order). |

Reviewer: ```suggestion
| `messageGroupId`          | string (optional) | The tag that specifies that a message belongs to a specific message group. Messages that belong to the same message group are processed in a FIFO manner. Messages in different message groups might be processed out of order. |
```

I think we can remove the parentheses here.
